### PR TITLE
Update to GNOME 40 runtime

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

**Comment by ccptr** [comment]
bot, build

**Comment by ccptr** [comment]
Seems to work for me. @nielsdg could you please merge it?

File: org.gnome.seahorse.Application.json [hunk 1]
```diff
@@ -117,31 +117,14 @@
...
-        },
         {
             "name": "seahorse",
             "buildsystem": "meson",
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/seahorse/3.38/seahorse-3.38.0.1.tar.xz",
-                    "sha256": "c745dd1de6e1a20f19a2b07a438b8ef7cca235a6aa73410db7f137b021e9b2a2"
+                    "url": "https://download-fallback.gnome.org/sources/seahorse/40/seahorse-40.0.tar.xz",
```
Comment (nielsdg) [inline org.gnome.seahorse.Application.json:126]:
```suggestion
                    "url": "https://download.gnome.org/sources/seahorse/40/seahorse-40.0.tar.xz",
```

**Review by nielsdg — COMMENTED** [review]
Sorry that I didn't see this earlier, my GitHub notifications are somehow acting off for the last few months.

Looks good, apart from the small remark. Thanks!

File: org.gnome.seahorse.Application.json (hunk 1 above)
Comment (A6GibKm) [reply org.gnome.seahorse.Application.json]: Done :)

**Pull request merged**